Sources/ComposedUI/CollectionView/CollectionCoordinator.swift

```diff
@@ -297,6 +304,28 @@ open class CollectionCoordinator: NSObject {
             logger(message)
         }
     }
+
+    fileprivate func dumpState() {
```

Comment:
May I know where do you invoke `dumpState()`?

Reply:
From the debugger when I add a breakpoint, or within a function during debugging (but not then committed)